jp/components/drop_down.md

```diff
@@ -0,0 +1,283 @@
+    <igx-drop-down (onOpening)="onOpening($event)">
+        <igx-drop-down-item *ngFor="let item of items">
+            {{ item.field }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+```
+
+```typescript
+// dropdown.component.ts
+
+...
+    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
+
+    public items: any[] = [];
+
+    constructor() {
+        for (let i = 1; i < 4; i ++) {
+            const item = { field: "Option " + i };
+            this.items.push(item);
+        }
+    }
+
+    public ngOnInit() {
+    }
+
+    public toggleDropDown() {
+        this.igxDropDown.toggle();
+    }
+
+    public onOpening(ev) {
+        this.igxDropDown.setSelectedItem(0);
+    }
+
+```
+
+<div class="sample-container" style="height: 200px">
+    <iframe id="dropdown-sample-2-iframe" src='{environment:demosBaseUrl}/dropdown-sample-2' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="dropdown-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">                stackblitz で開く
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+`isHeader` を使用して意味的なグループ化や `isDisabled` を使用して非インタラクティブな項目を表示するなど、有用なビジュアル情報を提供できます。
+
+```html
+<!-- dropdown.component.html -->
+
+<div class="sample-wrapper drop-down-wrapper">
+    <button igxButton="raised" (click)="toggleDropDown()">Countries</button>
+    <igx-drop-down>
+        <igx-drop-down-item *ngFor="let item of items" isDisabled={{item.disabled}} isHeader={{item.header}}>
+            {{ item.field }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+
+```
+
+```typescript
+// dropdown.component.ts
+
+...
+    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
+
+    public items: any[] = [
+        { field: "EU", header: true },
+        { field: "Germany" },
+        { field: "Bulgaria" },
+        { field: "UK", disabled: true }];
+
+    constructor() {
+    }
+
+    public ngOnInit() {
+    }
+
+    public toggleDropDown() {
+        this.igxDropDown.toggle();
+    }
+
+```
+
+サンプルが正しく構成されると、EU ヘッダーの下に国の一覧がグループ形式で表示され、UK は非インタラクティブな項目として表示されます。
+
+<div class="sample-container" style="height: 280px">
+    <iframe id="dropdown-sample-3-iframe" src='{environment:demosBaseUrl}/dropdown-sample-3' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dropdown-sample-3-iframe" data-demos-base-url="{environment:demosBaseUrl}">                stackblitz で開く
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+### API まとめ
+以下の表は、便利な **igx-drop-down**  コンポーネントの入力、出力、およびメソッドを示します。
+
+#### 入力
+以下の入力は **igx-drop-down** コンポーネントで利用できます。
+
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+| `width` | 文字列 | コントロールのタブ幅を設定します。 |
```

Comment:
Tab is probably wrong here.  Please remove it in this line and the next one.  This looks like a copy/paste error in EN.

Reply:
Removed 'tab' in JP.